packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -44,55 +44,39 @@ The SDK represents the sign-in flow using a number of different objects:
 </div>
 
 - **Response:** The top-level object that represents a step and contains all the other objects. It includes a property that indicates a successful sign-in and functions for cancelling the sign-in flow, or retrieving the access token after the sign-in flow succeeds. A response may contain multiple authenticators and remediations.
-- **Remediation:** Represents the main user actions for a step, such as enrolling in an authenticator or entering an OTP. It includes a function for requesting the next step in the flow.
+- **Remediation:** Represents the main user actions for a step, such as enrolling in an authenticator or entering an OTP. In the Swift SDK, it also includes the function for requesting the next step in the flow.
```

Comment:
@mikenachbaur-okta, lets review this functionality offline and see if this should be added to idx-android as well.